Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
  * SMTChecker: Support try/catch statements.
  * SMTChecker: Output internal and trusted external function calls in a counterexample's transaction trace.
  * SMTChecker: Synthesize untrusted functions called externally.
+ * Command Line Interface: Support equal sign "=" for ```--libraries``` option.
```

Comment:
```suggestion
 * Command Line Interface: Accept also equal sign "=" for ``--libraries``.
```

Reply:
Updated. Thanks @leonardoalt.

Reply:
Maybe we should be a bit more specific as to how it can be used? Something like `Allow `=` as separator between library name and address in --libraries commandline option`?

Reply:
Thanks @chriseth, updated.